hw03_phillipsj2.py

```diff
@@ -0,0 +1,209 @@
+######################################################################
+# Authors: Chase Phillips
+# Username: phillipsj2
+#
+# Assignment: HW03
+#
+# Purpose: A program that uses turtles to draw a complex object. This is also an intro to Git's more complex nature
+#
+#https://docs.google.com/document/d/1ScrNyDoxQ187_iHzXMwE0PJ0ZUmyG6eX1xUxiFIGk-Y/edit?usp=sharing
+######################################################################
+
+import turtle
+
```

Comment:
Check for gray squiggly lines. this one says put 2 blank lines before function definitions.